selfdrive/car/gm/interface.py

```diff
@@ -141,15 +141,15 @@ def get_params(candidate, fingerprint):
     ret.longPidDeadzoneBP = [0.]
     ret.longPidDeadzoneV = [0.]
 
-    ret.longitudinalKpBP = [5., 35.]
-    ret.longitudinalKpV = [2.4, 1.5]
-    ret.longitudinalKiBP = [0.]
-    ret.longitudinalKiV = [0.36]
+    ret.longitudinalKpBP = [0., 5., 35.]
+    ret.longitudinalKpV = [3.3, 2.425, 2.2]
+    ret.longitudinalKiBP = [0., 35.]
+    ret.longitudinalKiV = [0.36, 0.36]
 
     ret.steerLimitAlert = True
 
-    ret.stoppingControl = True
-    ret.startAccel = 0.8
+    ret.stoppingControl = False
```

Comment:
Do we still need to set it to False? With 0xb brake mode, stops feel pretty smooth already. Also, description of "stoppingControl" says "Does the car allows full control even at lows speeds when stopping", which _is_ the case for GM.